javascript/javascript1/week1/AliSharifi/index2.js

```diff
@@ -0,0 +1,17 @@
+function GoodboyOldboy(
+  dogYearOfBirth,
+  dogYearFuture,
+  shouldShowResultInDogYears
+) {
+  let dogYear = dogYearFuture - dogYearOfBirth;
+  if (shouldShowResultInDogYears) {
+    console.log(
+      `Your dog will be ${dogYear * 7} dog years old in ${dogYearFuture}`
+    );
+  } else {
+    console.log(
+      `Your dog will be ${dogYear} human years old in ${dogYearFuture}`
+    );
+  }
+}
+console.log(GoodboyOldboy(2017, 2027, true));
```

Comment:
Now you get one log from the function, and then one line that says `undefined`. To fix this, just implement the following.

```suggestion
GoodboyOldboy(2017, 2027, true);
```